src/agent/Bradley.kt

```diff
@@ -0,0 +1,24 @@
+package agent
+
+import action.ActionControl
+import brain.LearningUnit
+import brain.Memory
+import brain.VisualProcessingUnit
+import map.Node
+
+class Bradley {
+    var lifeUnits = 10
+    var currentNode =  Node(0,0)
+    companion object {
```

Comment:
Not sure, but this doesn't feel like something that should be shared between many Bradley's, but I don't know. I guess it depends.